source/aerobulk/flux.py

```diff
@@ -1,9 +1,34 @@
 import aerobulk.aerobulk.mod_aerobulk_wrap as aero
 
 
+def _check_shape(arr):
+    if arr.ndim != 3:
+        raise ValueError
+
+
 def flux_noskin(
-    sst, t_zt, hum_zt, u_zu, v_zu, slp=101000.0, algo="coare3p0", zt=10, zu=2, niter=1
+    sst,
+    t_zt,
+    hum_zt,
+    u_zu,
+    v_zu,
+    slp,
+    algo=None,
+    zt=None,
+    zu=None,
+    niter=None,
 ):
-    return aero.mod_aerobulk_wrapper.aerobulk_model_noskin(
+    # TODO check if removing default Nones here still passes kwargs down
+
+    for arg in [sst, t_zt, hum_zt, u_zu, v_zu, slp]:
+        _check_shape(arg)
+
+    # TODO is niter actually used by our fortran wrapper?
```

Comment:
@rabernat we were wondering about this. `Niter` is an input to the [fortran wrapper](    INTEGER,                  INTENT(in) :: Niter), but never gets passed. Am I missing something, or was this just an oversight?

Similarly [here](https://github.com/jbusecke/aerobulk-python/blob/8859536e0211726e4be3033023f6724c4cd8edcc/source/fortran/mod_aerobulk_wrap.f90#L30) I think we can delete that line?

Reply:
We should give the option to pass niter through.